src/olympia/versions/tests/test_models.py

```diff
@@ -869,6 +872,35 @@ def test_nomination_inherited_for_updates(self):
             amo.RELEASE_CHANNEL_LISTED, parsed_data=self.dummy_parsed_data)
         assert upload_version.nomination == pending_version.nomination
 
+    def test_doesnt_commit_to_git_by_default(self):
```

Comment:
nit: personally, I think it's easier to see what tests depend on waffles by adding `@override_switch('waffle-name-here', active=False)` to them even for `False` or when they're missing.